<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>764</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionShowFPS"/>
    <addaction name="actionShowAxis"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionFitScene"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
    <addaction name="actionLeftDock"/>
    <addaction name="actionRightDock"/>
   </widget>
   <addaction name="menu_Window"/>
   <addaction name="menuRendering"/>
   <addaction name="menuScene"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="leftDockWidget">
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>45</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="leftDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="leftDockVerticalLayout"/>
     </item>
     <item>
      <spacer name="leftDockVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="rightDockWidget">
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>45</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="rightDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
     <item>
      <layout class="QVBoxLayout" name="rightDockVerticalLayout"/>
     </item>
     <item>
      <spacer name="rightDockVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Show/hide fullscreen window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionShowFPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;FPS</string>
   </property>
   <property name="toolTip">
    <string>Show/hide FPS</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionShowAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Axis</string>
   </property>
   <property name="toolTip">
    <string>Show/hide axis</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionFitScene">
   <property name="text">
    <string>Fit scene</string>
   </property>
   <property name="toolTip">
    <string>Fit scene in order to show all drawable objects</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionLeftDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left dock</string>
   </property>
   <property name="toolTip">
    <string>Show/hide left dock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRightDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Right dock</string>
   </property>
   <property name="toolTip">
    <string>Show/hide right dock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
